portal/templates/assignments/view-assignments.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'layouts/base.html' %}
+
+{% block title %} Assignments for {{ assignments[0]['course_name'] }} {% endblock %}
+
+{% block content %}
+<form method='post'>
+  <h1>List of assignments</h1>
```

Comment:
<h1> is already used in base and since we extend that layout we are using it here so semantically we shouldn't have two h1's